curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f4add33ea4b1f61ba3dc8.md

```diff
@@ -0,0 +1,44 @@
+---
+id: 657f4add33ea4b1f61ba3dc8
+title: Step 22
+challengeType: 20
+dashedName: step-22
+---
+
+# --description--
+
+Finally try out the new implementation by executing the program. If you've done everythong properly, you should see the input string converted into a snake case like before.
```

Comment:
```suggestion
Finally try out this new implementation by executing the program. If you've done everything correctly, you should see the input string converted into snake case, like before.

Congratulations! Now your `convert_to_snake_case()` function is ready.
```
I would suggest adding a final message to congratulate learners on reaching the end of the project. 🙂